packages/components/pages/f-registration/src/components/Registration.vue

```diff
@@ -498,10 +498,19 @@ export default {
                         return;
                     }
 
-                    if (status === 403 || status === 401) {
+                    if (status === 401) {
                         this.$emit(EventNames.LoginBlocked);
                         return;
                     }
+
+                    if (status === 403) {
```

Comment:
LGTM, one ❓ - Is there any other reason why a 403 might be returned that won't contain MFS details, should it potentially check for the existence of MFA data in the error response?

Reply:
@tommcclean. No we've changed over the registration BFF to return a `401` for reCAPTCHA/Ravelin block. The only time a `403` will be returned now is when a MFA challenge is issued. The idea being that CoreWeb can hook into the event, get the token/target and redirect them to the new MFA page.

Reply:
Awesome, thanks